datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/policy/mappers/PolicyInfoPolicyMapper.java

```diff
@@ -53,6 +54,10 @@ private ActorFilter mapActors(final DataHubActorFilter actorFilter) {
     result.setAllGroups(actorFilter.isAllGroups());
     result.setAllUsers(actorFilter.isAllUsers());
     result.setResourceOwners(actorFilter.isResourceOwners());
+    UrnArray resourceOwnersTypes = actorFilter.getResourceOwnersTypesUrns();
```

Comment:
I'd prefer if we refer to this simply as "resourceOwnerTypes" since we do not use the "urns" suffix for the others here - users, groups, etc.